src/lib/components/FieldInput.svelte

```diff
@@ -47,7 +47,7 @@
 			id={name}
 			{disabled}
 			{placeholder}
-			type="text"
+			{...{ type }}
```

Comment:
Never seen this syntax, can you do `{type}` directly?

Reply:
I tried that first but I kept getting [this error](https://stackoverflow.com/questions/57392773/error-type-attribute-cannot-be-dynamic-if-input-uses-two-way-binding). I could only solve it by applying that syntax, which is described in [this reply](https://stackoverflow.com/questions/57392773/error-type-attribute-cannot-be-dynamic-if-input-uses-two-way-binding/75298645#75298645). I left the reference in the [commit](https://github.com/fmaclen/hollama/pull/202/commits/b332a85a0cac75ce638c9af003a775578f027297).

Reply:
Understood, let's add that reference as a comment in the code. I would have never looked at the commit comment 😅

Reply:
Don't forget to add this comment to the codebase